pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = --verbose --tb=short -n auto
```

Comment:
would it be possible to instead add a python dev-dependency on `pytest-xdist`? These tests can take a while to run, not running them in parallel would make it a lot slower

Reply:
reverted the `-n auto` change